packages/web-twig/src/Resources/partials/inputProps.twig

```diff
@@ -1 +1 @@
-{%- for propName, propValue in transferringAttributes  %} {{ propName }}="{{ propValue }}"{% endfor -%}
+{%- for propName, propValue in transferringAttributes  %}{%- if propValue is not null and propValue != '' %}{{ propName }}="{{ propValue }}"{% endif -%}{% endfor -%}
```

Comment:
Why not use stricter comparison, e. g. `!==`? In Twig there is an equivalent `not same as()` - https://twig.symfony.com/doc/2.x/tests/sameas.html.